compiler/rustc_target/src/spec/mod.rs

```diff
@@ -635,6 +635,10 @@ pub enum StackProbeType {
 }
 
 impl StackProbeType {
+    // LLVM X86 targets (ix86 and x86_64) can use inline-asm stack probes starting with LLVM 16.
+    // Notable past issues were rust#83139 (fixed in 14) and rust#84667 (fixed in 16).
```

Comment:
This is the reference I made to known issues, which were also mentioned in the distributed comments that I removed.